TESTING.md

```
@@ -0,0 +1,88 @@
+### Docker-based Testing
+
+For proper integration testing with MongoDB, use one of these approaches:
+
+#### 1. Using the run-tests.sh script
+
+```bash
+./run-tests.sh
+```
+
+This script will:
+- Check if Docker is available
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)